src/main/java/org/embl/mobie/viewer/plugins/platybrowser/GeneSearchCommand.java

```diff
@@ -0,0 +1,56 @@
+package org.embl.mobie.viewer.plugins.platybrowser;
+
+import bdv.util.BdvHandle;
+import bdv.viewer.SourceAndConverter;
+import ij.IJ;
+import net.imglib2.RealPoint;
+import org.embl.mobie.viewer.MoBIE;
+import org.embl.mobie.viewer.color.LabelConverter;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+import sc.fiji.bdvpg.scijava.ScijavaBdvDefaults;
+import sc.fiji.bdvpg.scijava.command.BdvPlaygroundActionCommand;
+
+import java.util.Arrays;
+
+import static de.embl.cba.bdv.utils.BdvUtils.getGlobalMouseCoordinates;
+
+@Plugin(type = BdvPlaygroundActionCommand.class, menuPath = "Platybrowser>ProSPr Gene Search")
+public class GeneSearchCommand implements BdvPlaygroundActionCommand
+{
+	@Parameter
+	BdvHandle bdvHandle;
+
+	@Parameter
+	SourceAndConverter[] sourceAndConverters;
+
+	@Parameter ( label = "Search radius [micrometer]")
+	private double micrometerRadius = 3.0;
+
+	private static MoBIE moBIE;
+
+	public static void setMoBIE( MoBIE moBIE )
+	{
+		GeneSearchCommand.moBIE = moBIE;
+	}
+
+	@Override
+	public void run()
+	{
+		double[] micrometerPosition = new double[ 3 ];
+		final RealPoint realPoint = new RealPoint( 3 );
+		bdvHandle.getViewerPanel().getGlobalMouseCoordinates( realPoint );
```

Comment:
Does this give you the right position? As you're selecting from the context menu, your mouse position will be different to where you originally clicked (especially since this is the last option in the menu)

Reply:
@K-Meech Amazing question! You really scared me there 😨 . However, it turns out, once the context menu pops up, the "focus" seems to be lost on the BDV window and thus the last position BDV remembers is the one where you clicked at 😅 .

To make sure and also for the user, I could add a BdvPointOverlay at the position where the user clicked for `GeneSearch`, what do you think? I think it would be nice.

Reply:
Yes - sounds good! Would be nice to show exactly where the gene search is coming from :smile:

Reply:
<img width="1238" alt="image" src="https://user-images.githubusercontent.com/2157566/152535216-e10778d6-82ae-45fd-b059-b54f9ed23d33.png">

In fact, this is really fun now. With the new S3 storage and some multi-threading optimizations on my side this is really usable even from home.

Any more suggestions or should I merge and release to MoBIE-beta?

Reply:
Looks cool! I'm seeing a different result on Windows though - for me, the circle appears at my mouse position after I select the 'search radius (micrometer)', not the original position. E.g. here I clicked at a point in the ventral nerve cord, and the circle appears much lower:
![Screenshot (389)](https://user-images.githubusercontent.com/24316371/152556830-af468c84-a4a2-4df0-8dd1-ac1a285b2285.png)